script/run.sh

```diff
@@ -31,7 +31,9 @@ fi
 
 
 # Setup volume mounts for compose config and context
-VOLUMES="-v $(pwd):$(pwd)"
+if [ $(pwd) != '/' ]; then
```

Comment:
There should be double quotes around `"$(pwd)"` otherwise spaces will break this I think.